### PR TITLE
feat(l1): assertoor CI integration

### LINKED ISSUE #920: [CD] Add job that runs a localnet with assertor on merge
If it fails, we should send an alert to slack.

Should we send something if it passes too?

### LINKED ISSUE #952: [CI] Add job that runs a localnet with assertor as part of the CI
This is an initial task related to assertoor integration, the idea is to have it running as part of the CI with the [`stability_check.yaml`](https://github.com/ethpandaops/assertoor-test/blob/master/assertoor-tests/stability-check.yaml) as defined in the [`assertoor_test`](https://github.com/ethpandaops/assertoor-test) repo. we'll need to strip off consensus related checks to be able to run the test quicker.

### PR DESCRIPTION
**Pull request by rodrigo-o**
**Motivation**

Assertoor integration as part of the CI

**Description**

This PR adds the [Assertoor GHA](https://github.com/ethpandaops/kurtosis-assertoor-github-action) to have a local devnet running with some checks as part of the CI. Most of the time for the stability checks was related to consensus, after removing those checks in a custom playbook created from [`stability-check.yaml`](https://github.com/ethpandaops/assertoor-test/blob/master/assertoor-tests/stability-check.yaml) we are able to run it as part of the CI. For assertoor to run with our [fork of ethereum-package](https://github.com/lambdaclass/ethereum-package) a small [commit](https://github.com/lambdaclass/ethereum-package/commit/811581ee8ab42639e4e9ef7c1197041689bfdfd1) was needed to the `ethereum-rust-integration` branch.

I've also added some entries to the `paths-ignore` on the push rules.

_Note about time: Most of the time spent in this new job is not the assertoor run, which takes ~3-5 mins, but the actual building of the docker image, which take ~9mins before starting the tests. This is done in parallel to the docker build in the CI, but now it's done 2 times. Fortunately, this is not the longest run, since `Hive / Cancun Engine tests (pull_request)` takes ~19m_

Resolves #952

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -75,6 +75,36 @@ jobs:
...
           load: true # Important for building without pushing
+          tags: lambda_ethereum_rust:latest
+          outputs: type=docker,dest=/tmp/lambda_ethereum_rust.tar
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: lambda_ethereum_rust
+          path: /tmp/lambda_ethereum_rust.tar
+
+  assertoor:
```
Comment (mpaulucci) [inline .github/workflows/ci.yaml:86]: can we move this to a different file?

File: test_data/el-stability-check.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+id: el-stability-check
```
Comment (mpaulucci) [inline test_data/el-stability-check.yml:1]: should we have this inside `.github` instead?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (rodrigo-o) [reply .github/workflows/ci.yaml:86]: Yeah for sure!

Comment (rodrigo-o) [reply .github/workflows/ci.yaml]: Done!

**Review by fkrause98 — APPROVED** [review]
LGTM!

**Pull request merged**